### DISCUSSION TITLE
CLI Command After Install

### DISCUSSION THREAD

**Discussion by codefoster** [microsoft/winget-cli#3233] [Q&A]
I just installed NVS (`jasongin.nvs`) via WinGet and expect to (after restarting PowerShell) be able to execute the `nvs` command on the command line, but I can't. Is there a way to inquire about a package about how to use it after it's installed?

**Comment by mdanish-kh** [comment]
I've tested by installing `jasongin.nvs` and the package is available in the PATH variable after installation, but you'll have to restart your terminal/shell process for the `$env:PATH` variable to be updated in your terminal session

- Related to https://github.com/microsoft/winget-cli/issues/3077
- Related discussion in https://github.com/microsoft/winget-cli/issues/549
